data/ships.txt

```diff
@@ -2804,11 +2802,11 @@ ship "Sparrow"
 	engine -5 35
 	engine 5 35
 	gun -7 -10 "Beam Laser"
-	gun 7 -10 "Meteor Missile Launcher"
+	gun 7 -10 "Beam Laser"
 	leak "flame" 60 80
 	explode "tiny explosion" 15
 	explode "small explosion" 5
-	description "Classified as an interceptor rather than a fighter because it has its own hyperdrive instead of needing to be carried inside a larger ship, the Tarazed Sparrow is the smallest and cheapest combat ship available. Because of its limited cargo and passenger space, the primary way for a Sparrow pilot to pay the bills is by hunting pirates... or by becoming one. In either case, it is a perilous way to earn a living."
+	description "Classified as an interceptor rather than a fighter because it has its own hyperdrive instead of needing to be carried inside a larger ship, the Tarazed Sparrow is the smallest and cheapest combat ship available. Despite having limited cargo space, the Sparrow is advertised as a maneuverable asteroid mining ship. However, many Sparrow pilots pay the bills by hunting pirates... or becoming one. In either case, it is a perilous way to earn a living."
```

Comment:
Sorry, one more proposed change to this sentence:

"However, many Sparrow pilots pay the bills by upgrading the weapons and hunting pirates... or becoming pirates themselves."